infrastructure/via/src/config.ts

```diff
@@ -180,8 +181,24 @@ export function pushConfig(environment?: string, diff?: string) {
     env.modify('DATABASE_MERKLE_TREE_BACKUP_PATH', `./db/${environment}/backups`, l2InitFile, false);
 
     env.reload();
+    await fetchCelestiaTrustedHash();
 }
 
+const fetchCelestiaTrustedHash = async () => {
```

Comment:
Maybe better to make this part of `via celestia` command, since we already have a logic there for getting the celestia auth_token and updating configs?

Reply:
We need to fetch the celestia Block hash before starting the containers because the block hash is injected through ENV. This needs to be done before running `make init`.